docs/references/third_party_packages_markdown.md

```diff
@@ -0,0 +1,86 @@
+Python 3rd-party Dependencies
+=============================
+
+Adding 3rd-party Packages
+-------------------------
+
+In many cases, it is useful to add 3rd-party Python packages that can be
+used in your plugin. To do this, add dependencies to the
+requirements-thirdparty.txt file in the tool workspace directory.
+
+There are 2 options:
+
+1.  Manually add dependencies. If you use this approach, make sure to
+    include all dependencies (including any sub-dependencies).
+
+2. Use `pip freeze > requirements.txt` to generate a new
+requirements.txt file in the workspace. You will need to prune this file
+to remove any dependencies that aren\'t explicitly imported or required
+by your plugin. Do this before you copy or overwrite the contents to the
+`requirements-thirdparty.txt` file
```

Comment:
TW review:

2. Use `pip freeze > requirements.txt` to generate a new
requirements.txt file in the workspace. You need to prune this file
to remove any dependencies that aren't explicitly imported or required
by your plugin. Do this before you copy or overwrite the contents to the
`requirements-thirdparty.txt` file.

Reply:
fixed